docs/testing.md

```diff
@@ -10,6 +10,7 @@ We highly recommend working in the style of [test-driven development](http://agi
 For our testing examples, we use [jest](https://facebook.github.io/jest/), but there are other options that can perform similar operations. We also recommend using [nock](https://github.com/nock/nock), a tool for mocking HTTP requests, which is often crucial to testing in Probot, considering how much of Probot depends on GitHub's APIs. Here's an example of creating an app instance and using nock to test that we correctly hit the GitHub API:
 
 ```js
+import {OctokitOptions} from "probot/lib/types";
```

Comment:
This is considered internal and shouldn't be relied upon as it can change at anytime without notice

Reply:
I see, thanks for the heads up.